docker-compose.yml

```diff
@@ -0,0 +1,26 @@
+
```

Comment:
stand up postgres container and performance manager container for local testing.